FilterViewController.m

```diff
@@ -27,11 +27,11 @@ - (void)viewDidLoad {
     MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Labels" tableView:self.tableView];
     
     //sends double array for filternames and single array for header lines
-    [labels setDataWithFilterNames:@[self.returnLabelArray] headerTitles:@[@"Labels"]];
+    [labels setDataWithFilterNames:self.returnLabelArray headerTitles:@[@"Labels"]];
     
     MHCollapsibleViewManager *surveys = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Surveys" tableView:self.tableView];
     
-    NSArray *surveyAnswers = @[@[self.returnSurveyArray], @[self.returnSurveyArray], @[self.returnSurveyArray]];
+    NSArray *surveyAnswers = @[self.returnSurveyArray, self.returnSurveyArray, self.returnSurveyArray];
```

Comment:
I was just trying to populate the table with data so I made each survey have the same entries (Question 1, 2, etc.), that's why the returnSurveyArray is returned 3 times creating the array of three arrays. Concatenating would mean all the entires in each array would be put into one, right? The inputs for a hierarchy need to be a double array. The manager takes an array of arrays so that each array will be the data for each survey.